packages/express/lib/circuit-breaker.js

```diff
@@ -0,0 +1,62 @@
+const { performance } = require('perf_hooks');
+
+// TODO: figure out good values
+const upperThreshold = 0.75;
+const lowerThreshold = 0.6;
```

Comment:
These values aren't tested yet, they're just a gut feeling. We need to deploy a sample app to our production cluster to fine-tune these values.
We should probably also make them configurable and provide sensible defaults.